src/test/java/com/verapdf/selenium/pages/BeforeTest.java

```diff
@@ -0,0 +1,29 @@
+package com.verapdf.selenium.pages;
+
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+import java.util.concurrent.TimeUnit;
+
+public class BeforeTest {
+    public WebDriver driver = new ChromeDriver();
+
+    @BeforeMethod
+    public void beforeStart() {
+        System.setProperty("webdriver.chrome.driver", "C:\\Users\\User\\Documents\\verapdf-webapp-selenium-tests\\chromedriver.exe");
```

Comment:
Do not use absolute paths, they depend on the user system. As for chrome driver, it is better to use driver manager or something like that to manage it (including download, that means no .exe file should be pushed). I guess you can consult with nadja.sych@duallab.com on how it is done in our project for example.